packages/grafana-ui/src/components/Section/SectionLabel.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+
+/**
+ * @alpha
+ */
+export const SectionLabel = ({ name, className }: { name: string; className?: string }) => (
+  <label className={cx('gf-form-label query-keyword', className)}>{name}</label>
```

Comment:
Is it possible to add a `for`attribute here?

Reply:
There's no single element it could point to. The label is for a row that contains multiple components, e.g. (stats, response, 500 are all separate segments):

<img width="296" alt="Screen Shot 2021-08-16 at 12 22 22" src="https://user-images.githubusercontent.com/745532/129549343-5b50c758-e184-4231-b56c-3ceb283ef868.png">

Reply:
Okay, in the next iteration it might be worth thinking about from an a11y point of view @Ijin08